packages/schematics/angular/secondary-entrypoint/schema.json

```
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "SchematicsLibrary",
+  "title": "Secondary Entrypoint Schema",
+  "type": "object",
+  "description": "Creates a secondary entrypoint in a library project in a project.",
+  "additionalProperties": false,
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "The name of the library.",
+      "pattern": "^(?:@[a-zA-Z0-9-*~][a-zA-Z0-9-*._~]*/)?[a-zA-Z0-9-~][a-zA-Z0-9-._~]*$",
```

The name should likely be just the name of the entry-point and not a combination of the library and entrypoint.

```
ng g library-entrypoint --project="@foo/bar" --name="buz"
```

```suggestion
      "pattern": "^[a-zA-Z0-9-._~]+/?[a-zA-Z0-9-._~]+$",
```